hcatalog/streaming/src/test/org/apache/hive/hcatalog/streaming/TestStreaming.java

```diff
@@ -436,6 +437,7 @@ public static void runWorker(HiveConf hiveConf) throws Exception {
   // stream data into streaming table with N buckets, then copy the data into another bucketed table
   // check if bucketing in both was done in the same way
   @Test
+  @Ignore
```

Comment:
Can you please create a follow-up jira to re-enable this test?

Reply:
@ashutoshc https://issues.apache.org/jira/browse/HIVE-21330 is follow-up jira.